torch/_torch_docs.py

```diff
@@ -469,6 +469,48 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.as_strided,
+           r"""
+as_strided(input, size, stride, storage_offset=0) -> Tensor
+
+Create a view of an existing `torch.Tensor` :attr:`input` with specified
+:attr:`size`, :attr:`stride` and :attr:`storage_offset`.
+
+.. warning::
+    :func:`as_strided` manipulates the internal data structure of tensors.
+    If done incorrectly, it can lead to the access of invalid memory and
```

Comment:
@SsnL says this isn't possible anymore. I played around with as_strided as well and noticed there are indeed checks, so maybe it isn't applicable anymore

Reply:
Ok. I will remove the first paragraph.